### DISCUSSION TITLE
facebook pixel

### DISCUSSION THREAD

**Discussion by 94tomas** [quasarframework/quasar#9653] [CLI - Capacitor / Cordova modes] [answered]
is there a way to integrate facebook pixel, and how effective is it to put the js code to index.html

**Comment by ontwikkelfabriek** [comment, marked as answer]
@94tomas 

>is there a way to integrate facebook pixel

Implementing it in Cordova/Capacitor will require a different way than to just put js in the 'index.html'(`index.template.html` btw for Quasar v1).  That's because unlike a SPA/website (created with Quasar), the webview in which your native mobile app( created in Quasar) is running, cannot be signed into facebook.

I would take a look at this cordova/capacitor plugin:
https://github.com/cordova-plugin-facebook-connect/cordova-plugin-facebook-connect

Here's a Vue github for using the FB pixel. But as stated before, probably not useful in a native mobile app created with cordova/capacitor.
https://github.com/threadhandler/vue-analytics-facebook-pixel

**Reply by 94tomas** [reply]
It is what I was afraid of, thanks for your answer I will look for more information from the link that happened to me

Greetings
